helix-plugin/README.md

```diff
@@ -0,0 +1,16 @@
+# Helix plugins system MVP
+
+This is still an early alpha, you should expect frequent breaking changes.
+The intend is to take inspiration from [CodeMirror 6](https://codemirror.net/6/docs/ref/).
+
+Plugins for Helix are written in any language that can be compiled to WASM (Web Assembly).
+This implies:
+- No language language-lockin, user can use what he's most confortable with to extend and customize his experience.
```

Comment:
Small nitpick: this should use `they're` and `their` instead of `he's` and `his`.